src/spacy/fortex/spacy/spacy_processors.py

```diff
@@ -575,23 +612,34 @@ def process_umls_entity_linking(linker, result, input_pack: DataPack):
 
     # get medical entity mentions and UMLS concepts
     for item in medical_entities:
-        entity = MedicalEntityMention(
-            input_pack, item.start_char, item.end_char
+        medical_entry_name = config.medical_onto_type
+        medical_entry = get_class(medical_entry_name)
+        entity = medical_entry(
+            pack=input_pack,
+            begin=item.start_char,
+            end=item.end_char,
         )
-        entity.ner_type = item.label_
+
+        setattr(entity, "ner_type", item.label_)
```

Comment:
I feel like these attribute names (ner_type) need to be in the config as well.

Reply:
I thought of doing exactly that initially, but then we will have to get at least 8 attribute names from config, 'ner_type' and 'umls_entities' for `MedicalEntityMentiona` dn 6 others for `UMLSConceptLink`. That was a lot of attribute names for config, so I changed it to the current code. Do you think its fine if we pass 8 diff attribute names from config?

Reply:
ok, maybe it is a bit ugly. Can be considered as a future fix if needed.